stable/kiam/values.yaml

```diff
@@ -11,7 +11,7 @@ agent:
 
   image:
     repository: quay.io/uswitch/kiam
-    tag: v2.8
+    tag: v3.0-rc1
```

Comment:
We discussed the issue of a release candidate version of the application in the charts call today. Here is the guidance:

For a release candidate version of software we want the chart version to also be a release candidate. The reason for this is that you need to specify that version to install it and we don't want to release (especially on stable) pre-release software by default. Helm will skip skip the pre-release RC like other forms of pre-release software. For example, to install this someone would need to use...

```
$ helm install stable/kiam --version 2.0.0-rc1
```

The same would apply to requirements files. There either a specific version or a range like `^2.0.0-0` (enables using pre-release versions) or `^2.0.0-rc1` (meaning equal to or newer than rc1).